c7n/resources/ec2.py

```diff
@@ -1517,7 +1518,19 @@ def process_instance(i):
 
 @actions.register('snapshot')
 class Snapshot(BaseAction):
-    """Snapshots volumes attached to an EC2 instance
+    """Snapshot the volumes attached to an EC2 instance.
+
+    Tags may be optionally added to the snapshot during creation.
+    - `copy-volume-tags` copies all the tags from the specified
```

Comment:
not sure exactly the reason, but sphinx isn't liking this doc string re build failure.
Warning, treated as error:
/home/runner/work/cloud-custodian/cloud-custodian/docs/source/aws/resources/ec2.rst:1712:Unexpected indentation.

Reply:
Thanks!  It was missing the blank line before the bulleted list.